package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "service-runner",
-  "version": "0.3.9",
+  "version": "0.4.0",
```

Comment:
Although, http://semver.org/#spec-item-4 basically says "anything goes" if major is < 1. 

I think we should move to 1.x soon to reflect that service-runner is now providing some reasonable stability guarantees.